tests/test_batch.py

```diff
@@ -86,3 +90,42 @@ def test_yaml():
     assert "OK" in ensset.keys()
     assert "npv.txt" in ensset.keys()
     assert not ensset.get_df("unsmry--yearly").empty
+
+
+def sleeper():
+    """Sleeps for one second.
+
+        This function must be a module member for it to be
+        pickled in concurrent applications"""
+    time.sleep(1)
+    return pd.DataFrame()
+
+
+def test_speedup():
+    """Naive test of speedup in concurrent mode"""
+
+    testdir = os.path.dirname(os.path.abspath(__file__))
+    ens = ScratchEnsemble(
+        "reektest", testdir + "/data/testensemble-reek001/" + "realization-*/iter-0"
+    )
+
+    set_concurrent(True)
+    # really_concurrent = use_concurrent()
+    start_time = datetime.datetime.now()
+    ens.process_batch(batch=[{"apply": {"callback": sleeper}}])
+    end_time = datetime.datetime.now()
+    conc_elapsed = (end_time - start_time).total_seconds()
```

Comment:
with time.time() you also wouldn't have to use total_seconds()
```suggestion
    conc_elapsed = (end_time - start_time).total_seconds()
```

Reply:
Yes